packages/node/src/config.ts

```diff
@@ -139,6 +140,20 @@ export class CloudDirectorConfig {
         }
     }
 
+    public makeLegacyApiClient() {
+        if (!this.connectionAuth.authorized) {
+            throw new Error("Connection not authorized, please login again to fix any auth errors.");
+        }
+
+        const username = this.authentication['username'];
+        const org = this.authentication['org'];
+        const token = this.authentication['authorizationKey'];
+
+        return new LegacyApiClient(this.basePath,
+            new CloudDirectorAuthentication(username, org, token),
```

Comment:
do you need a new instance of the authentication?

Reply:
Yes, because `this.authentication` is of an abstract type, which cannot be cast to `CloudDirectorAuthentication`